experimental/CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ All notable changes to experimental packages in this project will be documented
 
 ### :boom: Breaking Change
 
+* chore(sdk-node): deprecate methods in favor of constructor options [#3996](https://github.com/open-telemetry/opentelemetry-js/pull/3996) @pichlermarc
+  * The following methods are now deprecated and will be removed in a later release
```

Comment:
Should we pick a specific time or version?

Reply:
Good point. I'd do it in `0.43.0` (this change will appear in `0.42.0`). Added in [1534821](https://github.com/open-telemetry/opentelemetry-js/pull/3996/commits/1534821eacb156dcb71b15167b43d1ebc443c658)